orttraining/orttraining/python/training/ortmodule/__init__.py

```diff
@@ -9,6 +9,8 @@
 from glob import glob
 from packaging import version
 
+from ._fallback import _FallbackManager, ORTModuleInitException, wrap_exception
```

Comment:
>_FallbackManager [](http://example.com/codeflow?start=23&length=16)

and here

Reply:
This is a leftover during split commits with changes that addresses last review and what I am working in parallel.
Next commit will consume fallback manager or remove it in case I decide to add the current work on a second PR